batch_import/batch_download.py

```diff
@@ -168,8 +185,11 @@ def get_results_from_epoch(self, start_time, end_time, camera_names):
                         logging.debug("WARN - duplicate timestamps found, possible bug in iteration")
                     if not image.get('labels') or len(image['labels']) == 0:
                         continue
+                    new_labels = image['labels']
+                    if self.white_labels:
```

Comment:
TODO items is to make labels part of the query so that we fetch only the events of interest.